requirements-dev.txt

```diff
@@ -5,3 +5,4 @@ boto3
 coverage
 mock==3.0.5
 nose
+pynose
```

Comment:
Komt alleen hier voor? Is het nodig?

Reply:
package die er voor zorgt dat nose backwards compatible is